modules/ravestate_nlp/README.md

```diff
@@ -0,0 +1,96 @@
+```
+(C) Roboy 2019            
```

Comment:
```suggestion
```
Maybe remove the (C)... These will get outdated really fast (even keeping the on ein the root README up to date will be a challenge)